code/modules/mob/living/simple_animal/animal_defense.dm

```diff
@@ -79,6 +79,8 @@
 	. = ..()
 	if(.)
 		var/damage = rand(M.melee_damage_lower, M.melee_damage_upper)
+		if(MOB_EPIC in mob_biotypes && !(MOB_EPIC in M.mob_biotypes)) //epic mobs can still deal full damage to each other
+			damage *= 0.2
```

Comment:
Maybe, maybe, you should find a more specific way to deal with cheesy easy megafauna swarming (such as an appliable trait), rather than nerfing simplemobs as a whole.

Reply:
What would you suggest? People have opposed my idea of letting megafauna gib simplemobs around it in bigger quantities than one-by-one.

Reply:
the lowered damage thing is already good enough to slow down the horde of unstoppable killer tomatoes, but it should check if the attacker has such trait and the attacked's mob_biotypes includes MOB_EPIC.

Reply:
So... you mean for example adding a new biotype/trait/identifier/whatever for mobs usually used to cheese fauna?

Reply:
Yea, a trait (called like WEAK_ON_EPIC_MOBS or something) may be better.

Reply:
Done. Added a new biotype.